jiaptr.csl

```diff
@@ -0,0 +1,266 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US" page-range-format="expanded">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>JIAPTR(Journal of International Academy of Physical Therapy Research)</title>
+    <title-short>JIAPTR</title-short>
+    <id>http://www.zotero.org/styles/jiaptr</id>
+    <link href="http://www.zotero.org/styles/jiaptr" rel="self"/>
+    <link href="https://westlibrary.txwes.edu/sites/default/files/pdf/AMACitationStyle.pdf" rel="documentation"/>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issnl>2092-8475</issnl>
+    <summary>Same as the American Medical Association style as used in JAMA.</summary>
```

Comment:
@Jesse-Heo, so this style is the same as https://www.zotero.org/styles?q=id%3Aamerican-medical-association?

Reply:
yes, exactly, thank you for the help!

Reply:
I was working on JIAPTR's style  in the Zotero Style Repository(https://www.zotero.org/styles) to make it easy for researchers to understand the paper and it was very difficult for me because it was my first time.